src/coreclr/nativeaot/Runtime/eventpipe/ep-rt-aot.cpp

```diff
@@ -669,6 +669,46 @@ bool ep_rt_aot_spin_lock_release (ep_rt_spin_lock_handle_t *spin_lock)
     return false;
 }
 
+#ifndef HOST_WIN32
+#if defined(__APPLE__)
+#if defined (HOST_OSX)
+extern "C" {
+char ***_NSGetEnviron(void);
+}
```

Comment:
nit: I like the single line extern "C" function declaration, since there's only one

Reply:
Without the braces (but that is just me). Or indent if you prefer the multi-line with braces.